rita_client/src/heartbeat/mod.rs

```diff
@@ -92,9 +137,7 @@ pub fn send_udp_heartbeat() {
     }
 
     trace!("attempting to send heartbeat");
-    let dns_request = Resolver::from_registry()
-        .send(resolver::Resolve::host(heartbeat_url.to_string()))
-        .timeout(CLIENT_LOOP_TIMEOUT);
+    let dns_request = heartbeat_url.to_socket_addrs();
 
     // Check for the basics first, before doing any of the hard futures work
     #[allow(unused_assignments, unused_mut)]
```

Comment:
this flag for unused assignments and mut should be possible to remove now without a warning. is it?

Reply:
removed